corehq/motech/tests/test_value_source.py

```diff
@@ -1,12 +1,15 @@
 import doctest
-import warnings
+from datetime import datetime, timedelta
+from uuid import uuid4
 
-from django.test import SimpleTestCase
+from django.test import SimpleTestCase, TestCase
 
 import attr
 from schema import Use
 
 import corehq.motech.value_source
+from casexml.apps.case.models import CommCareCase
```

Comment:
This is a couch model. You may have heard that we're migrating away from that. :wink: Would these tests work if they used the SQL case model instead?

Reply:
> You may have heard that we're migrating away from that.

:joy: I'll switch them to use the SQL models.

Reply:
6dded0f